packages/docs/PropTables/shared.tsx

```diff
@@ -67,7 +67,7 @@ export const messagingLinkItemProps: Prop[] = [
     types: 'boolean',
     description: (
       <>
-        Shows an external icons when the <Code primary>external</Code> flag is set and target="_blank".
+        Shows an external icon when the <Code primary>external</Code> flag is set and target="_blank".
```

Comment:
Should we update `external flag` to `external prop`?

Reply:
Sounds like a good idea to me